<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programming Language Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="input_label">
      <property name="text">
       <string>Program Input:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="input_text">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="button_layout">
      <item row="0" column="0">
       <widget class="QPushButton" name="button_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="button_plus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="button_minus">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="button_times">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="button_divide">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_eq">
        <property name="text">
         <string>==</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_neq">
        <property name="text">
         <string>!=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="button_assign">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="button_lparen">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="button_rparen">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="button_quote">
        <property name="text">
         <string>"</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button_true">
        <property name="text">
         <string>true</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="button_false">
        <property name="text">
         <string>false</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="button_if">
        <property name="text">
         <string>if</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="button_then">
        <property name="text">
         <string>then</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="button_else">
        <property name="text">
         <string>else</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="button_end">
        <property name="text">
         <string>end</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="button_while">
        <property name="text">
         <string>while</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="button_do">
        <property name="text">
         <string>do</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="button_fun">
        <property name="text">
         <string>fun</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="button_return">
        <property name="text">
         <string>return</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="button_print">
        <property name="text">
         <string>print</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="button_clear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" columnspan="6">
       <widget class="QPushButton" name="button_equal">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="output_label">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="output_text">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="output_lcd">
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>9</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>